packaging/installer/methods/pfsense.md

```diff
@@ -18,6 +18,8 @@ learn_rel_path: "Installation/Install on specific environments"
 
 ## Install prerequisites/dependencies
 
+> **Note** The following instructions are meant to be used for the Commmunity Edition of pfSense. 
```

Comment:
```suggestion

```

Reply:
Cant's make a suggestion on the specific line (line 10), probably a bug, change the naming of this guide to _Install Netdata on pfSense CE_

Reply:
@tkatsoulas removed the added line and changed the title.